src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/SolutionView.xaml.cs

```diff
@@ -49,15 +50,36 @@ public SolutionView()
 
             _projectList.SizeChanged += ListView_SizeChanged;
 
+            //Requested Version column may not be needed, but since saved Sorting Settings are being restored at initialization time,
+            //we should go ahead and create the Header column for it here with its Sort property name.
+            _requestedVersionColumn = new GridViewColumnHeader();
+            SortableColumnHeaderAttachedProperties.SetSortPropertyName(_requestedVersionColumn, "RequestedVersion");
+
             _sortableColumns = new List<GridViewColumnHeader>
             {
                 _projectColumnHeader,
-                _versionColumnHeader
+                _installedVersionColumnHeader,
+                _requestedVersionColumn
             };
 
             SortByColumn(_projectColumnHeader);
         }
 
+        private void SolutionView_DataContextChanged(object sender, DependencyPropertyChangedEventArgs e)
+        {
+            //Since this event will fire when closing PMUI, when the model is set to Project PMUI,
+            //and when other WPF controls in the tree set their DataContext, this cast is more important than it may seem.
+            var model = e.NewValue as PackageSolutionDetailControlModel;
```

Comment:
No need to change it, but `is` has an optimization so you can write something like:

```cs
if ( e.NewValue is PackageSolutionDetailControlModel model)
```

Reply:
Did it since was making other changes